app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -79,7 +79,7 @@ fun ButtonBar() {
 @Composable
 fun Menu() {
     val items = listOf("A", "B", "C", "D", "E", "F")
-    var countryName : String by remember { mutableStateOf("Menu Categories") }
+    var categoryName : String by remember { mutableStateOf("Menu Categories") }
```

Comment:
A similar change was covered in a previous review.  When that occurs, first in wins.